### PR TITLE
Add telephone icon to social menu

### LINKED ISSUE #843: Social Menu: Add telephone icon
Add icon for `tel` in URLs.

### PR DESCRIPTION
**Pull request by laurelfulford**
### All Submissions:

* [x] Have you followed the [Newspack Contributing guideline](https://github.com/Automattic/newspack-theme/blob/master/.github/CONTRIBUTING.md)?
* [x] Does your code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) and [VIP Go coding standards](https://vip.wordpress.com/documentation/vip-go/code-review-blockers-warnings-notices/)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?

### Changes proposed in this Pull Request:

This PR adds a smartphone icon to the social links menu, so when you add a link to a phone number (prefixing it with `tel:`), it will display as a smartphone.

Closes #843

### How to test the changes in this Pull Request:

1. Apply the PR.
2. Add a telephone number to your social links menu, prefixing with `tel:`
3. Confirm that it displays as a smartphone:

image

### Other information:

* [x] Have you added an explanation of what your changes do and why you'd like us to include them?
* [ ] Have you written new tests for your changes, as applicable?
* [ ] Have you successfully ran tests with your changes locally?

### CODE REVIEW

File: newspack-theme/classes/class-newspack-svg-icons.php [hunk 1]
```diff
@@ -415,6 +417,11 @@ public static function get_social_link_svg( $uri, $size ) {
...
 </svg>',
 
+		'smartphone'  => /* material-design - smartphone */ '
```
Comment (thomasguillot) [inline newspack-theme/classes/class-newspack-svg-icons.php]: This PR works for me but the "Smartphone" icon is a bit hard to understand and almost too "abstract".

I'd suggest we use the "Phone" icon instead:

```
<svg xmlns="http://www.w3.org/2000/svg" height="24" viewBox="0 0 24 24" width="24"><path d="M20.01 15.38c-1.23 0-2.42-.2-3.53-.56-.35-.12-.74-.03-1.01.24l-1.57 1.97c-2.83-1.35-5.48-3.9-6.89-6.83l1.95-1.66c.27-.28.35-.67.24-1.02-.37-1.11-.56-2.3-.56-3.53 0-.54-.45-.99-.99-.99H4.19C3.65 3 3 3.24 3 3.99 3 13.28 10.73 21 20.01 21c.71 0 .99-.63.99-1.18v-3.45c0-.54-.45-.99-.99-.99z"/></svg>
```

Comment (laurelfulford) [reply newspack-theme/classes/class-newspack-svg-icons.php]: Good call, Thomas! I was a little worried about it looking archaic, but it just looks like a box inside of another box :) 

The icon has been updated in 2ac5c45.

**Comment by laurelfulford** [comment]
The icon has been updated and is ready for a second review! Thanks for the eyes, Thomas!

**Comment by laurelfulford** [comment]
Thanks!

**Pull request merged**